specification/ilm/start/StartIlmRequest.ts

```diff
@@ -29,9 +28,4 @@ import { Duration } from '@_types/Time'
  * @availability stack since=6.6.0 stability=stable
  * @cluster_privileges manage_ilm
  */
-export interface Request extends RequestBase {
-  query_parameters: {
-    master_timeout?: Duration
-    timeout?: Duration
-  }
-}
+export interface Request extends RequestBase {}
```

Comment:
Those parameters are supported and should be documented too: https://github.com/elastic/elasticsearch/blob/ce7a0c86fb975f76dfbea5eff1cf51602dbed184/x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/ilm/action/RestStartILMAction.java#L37